docs/users.rst

```diff
@@ -900,30 +900,37 @@ This example uses RHEL 8.4 EUS's Pulp repository IDs:
   - rhel-8-for-s390x-appstream-eus-rpms__8_DOT_4
 
 
-Using Artifacts from Koji
+Using Artifacts from Koji or Project Newcastle
 -------------------------
 
 During a container build, it might be desirable to fetch some artifacts
-from an existing Koji build. For instance, when building a Java-based
-container, JAR archives from a Koji build are required to be added to
-the resulting container image.
+from an existing Koji build or a Project Newcastle build. For instance, when building
+a Java-based container, JAR archives from a Koji build or PNC build are required
```

Comment:
You should define what PNC means before. Can we have also link to PNC webpage included somewhere?

Reply:
Updated. At first, mention of Project Newcastle, I've added (aka PNC) to clarify.